api/osrd_infra/views/rolling_stock.py

```diff
@@ -27,3 +56,14 @@ class LightRollingStockView(
     queryset = RollingStock.objects.order_by("id")
     serializer_class = LightRollingStockSerializer
     pagination_class = CustomPageNumberPagination
+
+
+class RollingStockLiveryView(ViewSet):
+    # peut-être supprimer le serializer + transaction atomique
```

Comment:
:baguette_bread:

Reply:
I would move all the logic (image composition) here. You should keep the serializer just to check the validity of the payload.